### PR TITLE
Update ballerina to bal in cli-help text

### LINKED ISSUE #28180: Ballerina CLI man-page still shows `ballerina` command instead of `bal`
**Description:**
When I type a wrong command (i.e bal run nonExistingFile), still the CLI help shows the following. It should be `bal run` instead of `ballerina run`
```
USAGE:
    ballerina run [--experimental] [--offline] [--taint-check]
```

**Steps to reproduce:**
```
bal run nonExistingFile
```

**Affected Versions:**
alpha1

**OS, DB, other environment details and versions:**
n/a

### PR DESCRIPTION
**Pull request by suganyasuven**
Fixes #28180 and https://github.com/ballerina-platform/ballerina-update-tool/issues/71

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help [hunk 1]
```diff
@@ -1,5 +1,5 @@
 NAME
-       The bal build tool
+       The ballerina update tool
```
Comment (grainier) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help:2]: Is this correct? Shouldn't this be `ballerina build` tool?

Comment (suganyasuven) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help:2]: Noted. This is on the discussion. Not Finalized yet. https://github.com/ballerina-platform/ballerina-update-tool/issues/71. Shall we check with others as well, please?

Comment (keizer619) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help:2]: Its not just update tool or build tool, Lets change it to `The Ballerina tool`

**Review by praneesha — APPROVED** [review]

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help (hunk 1 above)
Comment (Kishanthan) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help]: Ballerina with the uppercase B

Comment (suganyasuven) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help]: Noted.

**Pull request closed without merging**